ArmaforcesMissionBot/Modules/Signups.cs

```diff
@@ -685,6 +685,31 @@ public async Task EditMission(int missionNo)
             }
         }
 
+        [Command("edytuj-nazwe-misji")]
+        [Summary("Edycja nazwy już utworzonej misji.")]
+        [ContextDMOrChannel]
+        public async Task MissionName([Remainder] string newTitle)
+        {
+            var signups = _map.GetService<SignupsData>();
+
+            if (signups.Missions.Any(x =>
+                (x.Editing == ArmaforcesMissionBotSharedClasses.Mission.EditEnum.Started) &&
+                x.Owner == Context.User.Id))
```

Comment:
Wouldn't it be better to create some helper method on the service? Something like `GetCurrentlyEditedMission(userId)`, then you can if it's not not null in the if and also use it below.

Reply:
Yeah, it would be for sure. It's probably the most common 3-liner in this app, although I think it's out of scope of this little feature and will be done in ongoing signups refactor.

Reply:
I've just copied it from other parts. There are 11 more occurrences of this exact code. Do you want me to create this method and replace it everywhere?

Reply:
No, leave it for now. We will change this in #29.